src/TemplateHelpers.hs

```diff
@@ -184,6 +178,26 @@ docLinkRoute D.LinksContext{..} srcModule link = case D.linkLocation link of
       (PathVersion version)
       (P.runModuleName modName)
 
+getDepHtmlRenderContexts
+    :: D.Package a
+    -> Handler (Map Bower.PackageName (P.ModuleName -> HtmlRenderContext))
```

Comment:
I think it might be a bit neater to move the `Map.lookup` inside here so that this function returns a `Handler (Bower.PackageName -> P.ModuleName -> Maybe HtmlRenderContext`